ContactManager/ContactManager/Contacts/ConsoleInterface.cs

```diff
@@ -0,0 +1,116 @@
+﻿namespace ContactManager.Contacts;
+
+/// <summary>
+/// The ConsoleInterface Class is with wich the user interacts
+/// </summary>
+public class ConsoleInterface
+{   
+    
+    private ContactManagerFunctionality _ContactManager;
+    private Validator _Validator;
+
+    /// <summary>
+    /// Dependency Injection Through Contructor of ConsoleInterface Class
+    /// </summary>
+    public ConsoleInterface(ContactManagerFunctionality i, Validator validator)
+    {
+        _ContactManager = i;
+        _Validator = validator;
+    }
+
+    /// <summary>
+    /// Displays a Menu With List of operations available and promts user for a Choice
+    /// </summary>
+    /// <returns> Input of User as integer value</returns>
+    public int Menu()
+    {
+        Console.WriteLine("\nEnter the Operation You want to perform: \n [1] - Add Contact \n [2] - Display Contacts \n [3] - Delete \n [4] - Update \n [5] - exit \n [6] - Search \n [7] - Sorted View\n");
+        var UserInput = Console.ReadLine();
```

Comment:
camelCase for variables

```suggestion
        var userInput = Console.ReadLine();
```